### DISCUSSION TITLE
How to get trace result with verifyta?

### DISCUSSION THREAD

**Discussion by m5261136** [UPPAALModelChecker/UPPAAL-Meta#214] [Q&A] [answered]
How can I get trace result using verifyta?
When I use the command below, no files are generated and I get verification results but no trace results.

verifyta -f output testcase.xml queryfile
verifyta -y output testcase.xml queryfile

When I verified with uppaal, I obtained trace results successfully.

**Comment by thorulf4** [comment, marked as answer]
You need to specify that you want to generate a trace. This is done with the --diagnostic ( -t for short) see help for more information on how to use this option.

Try running with `verifyta -t 0 testcase.xml queryfile`
You can then redirect the trace output with `-f` option
